client/src/routes/Sessions/hooks/useSessions.ts

```diff
@@ -30,13 +30,14 @@ const useSessions = () => {
       startTime: dayjs.Dayjs;
       language: Session['language'];
     }) => {
-      await sessionApi.addSession({
+      const session = await sessionApi.addSession({
         contentId,
         type,
         startTime: startTime.toJSON(),
         language,
       });
       fetchSessions();
+      return session;
```

Comment:
May I suggest we test this too? :)

Reply:
Yes! great catch/reminder

Reply:
[7acc99a](https://github.com/29ki/29k/pull/848/commits/7acc99a816f13e4e97467bf91c95b91f2bd04400) ✅